app/src/main/java/com/hous/hous_aos/ui/rules/RulesViewModel.kt

```diff
@@ -159,6 +160,28 @@ class RulesViewModel @Inject constructor(
         }
     }
 
+    /** put
+     * 나의 to-do check한 거 보내기
+     * */
+    fun setMyToDoCheckBoxSelected(position: Int) {
+        val isSelected = myTodoList.value!![position].isChecked
+        myTodoList.value!![position].isChecked = !isSelected
+        val checked = myTodoList.value!![position].isChecked
+        val id = myTodoList.value!![position].id
+        viewModelScope.launch {
+            rulesTodayRepository.putMyToDoCheckLust("", id, MyToDoCheckRequest(checked))
+                .onSuccess {
```

Comment:
```suggestion
        viewModelScope.launch {
            rulesTodayRepository.putMyToDoCheckLust("", id, MyToDoCheckRequest(!myList.value!![position].isChecked))
                .onSuccess {
```
위에 isSelected, checked 삭제하고 이렇게 한줄로 해도 되지 않나요??

Reply:
제가 나중에 볼때 헷갈려서 지역 변수를 달았습니당  
없애는게 좋을까요??